tests/ipv6-evasion/ipv6-dos-slaac-advertises/README.md

```diff
@@ -0,0 +1,7 @@
+# Description
+
+Detect an attack that disrupts SLAAC by always replying with advertises and therefore prevents new hosts from obtaining an IP.
+
```

Comment:
This seems like ARP spoofing.
I do not know how to detect this without context, ie what is the difference between a legitimate response and a spoofed one

Reply:
I agree that it's rather hard to tell. We could look at what they did in the paper with snort3 as they mention snort3 with the patch detects it.